playbooks/roles/ext/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+# This playbook installs Zowe extensions on an existing Zowe instance
+
+# ============================================================================
+# validate variables
+- import_role:
+    name: common
+    tasks_from: validate_variables
+  vars:
+    variable_list:
+    - zowe_root_dir
+    - zowe_instance_dir
+    - zowe_extension_dir
+    - zowe_logs_dir
+
+# ============================================================================
+# Download Extension
+- import_role:
+    name: ext
+    tasks_from: upload-ext
+
+# ============================================================================
+# Install Extensions onto Zowe
+- name: Install Zowe Extension
+  import_role:
+    name: zos
+    tasks_from: run_script
+  vars:
+    script_chdir: "{{ zowe_root_dir }}/bin"
+    script_filename: ./zowe-install-component.sh
+    script_parameters: "-d \"{{ zowe_extension_dir }}\" -i \"{{ zowe_instance_dir }}\" -o \"{{ work_dir_remote }}/{{ filename_to_upload }}\" -l \"{{ zowe_logs_dir }}\""
```

Comment:
Think `zowe_extension_filename` variable is better than `filename_to_upload`.

Reply:
You're right, it makes more sense.